solr/test-framework/src/java/org/apache/solr/cloud/api/collections/AbstractIncrementalBackupTest.java

```diff
@@ -510,11 +510,7 @@ private void assertFolderAreSame(URI uri1, URI uri2) throws IOException {
             Arrays.sort(files1);
             Arrays.sort(files2);
 
-            try {
-                assertArrayEquals(files1, files2);
-            } catch (AssertionError e) {
-                e.printStackTrace();
-            }
+            assertArrayEquals(files1, files2);
```

Comment:
Does this fail occasionally?

Reply:
I hope not, but who knows? If it starts failing then we can at least add a comment or some explanation. Otherwise that seems pretty mysterious to me.